src/_data/sidenav.yml

```diff
@@ -1,38 +1,65 @@
-- title: Get started
+- header: Get started
+- title: Installation
   children:
     - title: Install Flutter
       permalink: /get-started/install
     - title: Test drive
       permalink: /get-started/test-drive
```

Comment:
Since the next section is called "Learn Flutter", it would be nice if this section also started with a bare infinitive. 

How about we do something like the following:

1. Drop the "Test drive" entry (as it's mostly duplicated content)
2. Make install a top-level entry with no children
3. Rename "Install Flutter" to "Set up Flutter"

Reply:
💯  I really dislike the "test drive" page and especially dislike that it's called "Test drive".